docs/plugins/image.md

```diff
@@ -1,77 +1,81 @@
 # Image
 
-The standard way of displaying images in Lightning is to just specify the `src`. This is the prefered way for your App's local assets (such as background, splash screen, logo and icons).
+The standard way of displaying images in Lightning is to just specify the `src`. This is the preferred way for your local assets (such as background, splash screen, logo and icons) of Lightning Apps.
 
-It's adviced that you optimize an App's local assets, by resizing them to the exact size and quality you will be using them. This will be beneficial for the memory usage of your App.
+It is recommended that you *optimize* the local assets of your App by resizing them to the *exact* size and quality in which you will use them. This positively affects the memory usage of your App.
 
-However when you don't have control over the images you're displaying in your App (because they come from a remote API for example), you can use the Image plugin to resize and crop images.
+However, if you don't have control over the images to be displayed in your App (for example, because they originate from a remote API), you can use the *Image* plugin to resize and crop them.
 
 ## Usage
 
-Whenever you need to resize image, import the Image plugin from the Lightning SDK
+Import the Image plugin from the Lightning SDK in components where you want to resize an image:
 
 ```js
 import { Img } from '@lightningjs/sdk'
 ```
 
-## Available methods
+## Available Methods
 
-### Exact
+### exact()
 
-Resizes the image to the exact dimensions, ignorning the ratio.
+
+Resizes the image to the exact dimensions, ignoring the ratio.
 
 ```js
 Img(url).exact(width, height)
 ```
 
-### Landscape
+### landscape()
 
 Resizes the image by width, maintaining the ratio.
 
 ```js
 Img(url).landscape(width)
 ```
 
-### Portrait
+### portrait()
 
-Resizes the image by height, maintaining the ratio
+Resizes the image by height, maintaining the ratio.
 
 ```js
 Img(url).portrait(height)
 ```
 
-### Cover
+### cover()
 
-Resizes the image in such a way that it covers the entire area. Depending on the orientation (portrait or landscape) of the image it will resize the image by width or by height.
+Resizes the image in such a way that it covers the entire area. Depending on the orientation (portrait or landscape) of the image and the target, it resizes the image by width or by height.
```

Comment:
not sure if 'target' is clear enough, maybe should be something like "Depending on the orientation (portrait or landscape) of the source image and the orientation of the final result"

Reply:
Suggestion: "Depending on the orientation (...) of the source image and that of the final result, it [...]."

Reply:
maybe we should say "the desired output" instead of "the final result" ?

Reply:
You suggest the following: "Depending on the orientation (...) of the source image and that of the desired output, it [...]."

I think 'desired output' instead of 'final result' is fine, since it covers the meaning of the 'result' in a more appropriate manner. Note: this will be adjusted for both the 'cover' and 'contain' methods.